recipe.cake

```diff
@@ -0,0 +1,66 @@
+#load nuget:https://hermes.chocolatey.org:8443/repository/choco-internal-testing/?package=cake-chocolatey-recipe&prerelease&version=0.1.0-unstable0082
+
+
+///////////////////////////////////////////////////////////////////////////////
+// ADDINS
+///////////////////////////////////////////////////////////////////////////////
+
+// (None)
+
+///////////////////////////////////////////////////////////////////////////////
+// RECIPE SETUP
+///////////////////////////////////////////////////////////////////////////////
+
+
+Environment.SetVariableNames();
+var packageSources = new List<PackageSourceData>();
+var nugetDevUrl = EnvironmentVariable("NUGET_DEV_SOURCE");
+var nugetProdUrl = EnvironmentVariable("NUGET_PROD_SOURCE");
```

Comment:
Just flagging these as possibly being change to `NUGETDEV_SOURCE` if you decide to go that route.

Reply:
think I'd need to remove one of them or release builds would fail since it tries to push to both on release builds as I understand it, but yep, thanks!